AzureMonitorAgent/agent.py

```diff
@@ -2229,7 +2229,7 @@ def run_get_output(cmd, chk_err = False, log_cmd = True):
             exit_code = e.returncode
             output = e.output
 
-    if type(output) == str:
+    if all(ord(c) < 128 for c in output):
```

Comment:
what about doing `errors='replace'` instead of checking the input?

Reply:
this works in python3 but not python2:
 txt = txt.encode("utf-8", errors='replace')
UnicodeDecodeError: 'ascii' codec can't decode byte 0xc3 in position 13: ordinal not in range(128)

https://onecompiler.com/python2/42qmtwtm5